kratos/python/add_vector_to_python.cpp

```diff
@@ -455,6 +451,50 @@ void  AddVectorToPython(pybind11::module& m)
     py::implicitly_convertible<py::list, Vector>();
     py::implicitly_convertible<array_1d<double,3>, Vector>();
 
+    //***********************************************************************************
+        //***********************************************************************************
+    py::class_< DenseVector<bool>, std::shared_ptr<DenseVector<bool>> > bool_vector_binder(m,"BoolVector", py::buffer_protocol());
```

Comment:
I was going to add some constexpr to use the already existing definitions and minimize code duplication

Reply:
well, the vector interface could be definitely be improved...however i would do it in a different Pr. What do you think?